azure-pipelines.yml

```diff
@@ -606,7 +597,7 @@ jobs:
                  -e CLI_VERSION=$CLI_VERSION \
                  -e HOMEBREW_UPSTREAM_URL=$HOMEBREW_UPSTREAM_URL \
                  --name azurecli \
-                 mcr.microsoft.com/mirror/docker/library/python:3.11-slim \
+                 mcr.microsoft.com/azurelinux/base/python:3 \
```

Comment:
Formula generation is independent of the Python version.
Use Azure Linux's 3.12 to avoid updates for the next four years.